CHANGELOG.md

```diff
@@ -6,8 +6,8 @@
 * [#785](https://github.com/ruby-grape/grape-swagger/pull/785): Add extensions for params - [@MaximeRDY](https://github.com/MaximeRDY).
 * [#782](https://github.com/ruby-grape/grape-swagger/pull/782): Allow passing class name as string for rake task initializer - [@misdoro](https://github.com/misdoro).
 
-
 #### Fixes
+* [#786](https://github.com/ruby-grape/grape-swagger/pull/786): Use full entity name as a default - [@mrexox](https://github.com/mrexox)
```

Comment:
would say its a feature insead of a fix 😉

Reply:
Yep! Moved a line 👌